scalableminds/webknossos
# Sandbox tracing mode (allow editing but don't allow saving)

This PR adds a (hidden) sandbox mode to tracings (available via the GET parameter `?sandbox`). For a public tracing (empty as well as not empty), we can use this from within the connectome viewer to display a tracing, which can be edited. Not sure, whether we want to make this feature even more accessible for not registered users?

@daniel-wer Right now, the GET parameter is read in the model initialization and then the annotation object is modified accordingly. Doesn't feel super clean to me, but I'm still looking for a better alternative. Do you have an idea?

### URL of deployed dev instance (used for testing):
- https://anonymoustracing.webknossos.xyz/

### Steps to test:
- demo: https://anonymoustracing.webknossos.xyz/annotations/Explorational/5dad58850100007500733944?sandbox&token=8m_-ghUs-Nk2LqEnrfMY0Q#4845,4961,2176,0,0.527,4
- append `?sandbox` to a tracing url and open it
- ensure that you can edit the tracing but that updates are/can not be saved


### Issues:
- contributes to #3520

------
